docs/book/src/quick-start.md

```diff
@@ -180,7 +180,7 @@ If you pressed `y` for Create Resource [y/n] then you created a CR for your CRD
 API definition):
 
 ```bash
-kubectl apply -k config/samples/
+kubectl apply -f config/samples/webapp_v1_guestbook.yaml
```

Comment:
Could you please let us know more about you motivation for this change?

Reply:
Ah, I just noticed I was too quick to open a PR and did not see that this is already fixed here (`-k` as seen above vs  `-f` in the book). My bad!

Does the book only get updated on releases?